ImageSharp/Program.cs

```diff
@@ -1,38 +1,29 @@
 ﻿using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.Formats.Png;
 using SixLabors.ImageSharp.PixelFormats;
 using SixLabors.ImageSharp.Processing;
 
-// Create an empty image canvas that is 256 x 256 px by ImageSharp.
-using var canvas = new Image<Rgba32>(256, 256);
+// Create a canvas from the table image.
+using var canvas = await LoadImage("./assets/table.png");
 
-// Load the table image and draw it on the canvas
-using var tableImage = await LoadImage("./assets/table.png");
-canvas.Mutate(ctx => ctx.DrawImage(tableImage, 1f));
+// Clear out any preexisting color palette if the image has one.
+// The encoder will automatically create a new palette if needed.
+PngMetadata metadata = canvas.Metadata.GetPngMetadata();
```

Comment:
Since the image contains a color table, we need to remove it to allow the library to generate a new one. This is because the apple image we are drawing is made up of colors not present in the existing palette.